packages/vue/src/components/verify-user.vue

```diff
@@ -87,6 +84,7 @@ const onSkipClicked = (): void => {
           <base-wrapper
             class="amplify-flex amplify-field amplify-radiogroupfield amplify-authenticator__column"
           >
+            <!-- TODO(BREAKING): remove hard coded string 493c -->
             <base-label
               class="amplify-visually-hidden amplify-label"
               id="amplify-field-493c"
```

Comment:
We should use utils such as `nanoid` to get unique ids. Left as-is because such change would be breaking.